### ISSUE TITLE
Overfitting on waymo data set?

### ISSUE THREAD

**Issue by mariya12290** [open-mmlab/mmdetection#5167]
Hey Admin and all,

I am trying to train Second on waymo data set, but model is getting over fit on waymo data set. Used hyper parameters and model architecture are both same as for kitti data set.

But from  observation, waymo data set is sequence, where as kitti is random. So I randomly feeding the data into the network in both data set cases. Still model is overfitting just on waymo.  

I am doing the batch normalization, instead of dropout and regularization.   From my understanding, batch normalization gives same effect as regularization. 

More info about data set and model:

- Size of data set is almost same around 6k in both kitti and waymo?
- But, target instance number is different, 5k in kitti and 30k in waymo. 800 and 5k in valid and test respectively. 

Here target is pedestrian. 

Could someone help me with this problem?

Thanks in advance

**Comment by shinya7y** [comment]
> Second

Here is not the mmdetection3d repository.

**Comment by mariya12290** [comment]
Thank you so much.
